docs/docusaurus.config.js

```diff
@@ -252,13 +252,6 @@ const config = {
             ],
             to: "/main/modules/bank",
           },
-          {
-            from: [
```

Comment:
As Marko mentioned, when IBC has a link about capability's readme (after https://github.com/cosmos/ibc-go/blob/marko/capability/capability/README.md is merged). We should redirect these links and `/main/modules/capability` to the readme.